helm/external-secrets/README.md

```diff
@@ -0,0 +1,182 @@
+# External Secrets
```

Comment:
I think we used to delete this and the template because they are not valid as we add / change some values to be giantswarm specific and this does not reflect the change.